api/google-manager/services/google-manager.js

```diff
@@ -1,3 +1,4 @@
+const { CronJob } = require('cron');
 const fs = require('fs');
 const { JWT } = require('google-auth-library');
```

Comment:
Is JWT a library, if it is, what does new JWT() mean?

Reply:
It is a class in the google-auth-library https://cloud.google.com/nodejs/docs/reference/google-auth-library/latest/google-auth-library/jwt.